schemas/infoAssessment.json

```diff
@@ -116,7 +116,8 @@
                 "password": {
                     "description": "Password to begin the assessment (only for Exams).",
                     "type": "string"
-                }
+                },
+                "SEBconfig": { }
```

Comment:
Could make this:
"SEBConfig": {
    "description": "...",
    "type": "object"
}